### ISSUE TITLE
[Bug]: Can't create multiple items by dragging a list any more

### ISSUE THREAD

**Issue by pjeby** [mgmeyers/obsidian-kanban#315]
### Describe the bug

Dragging and dropping a list onto the add item box creates a single card with multiple lines, instead of one item per card, making it harder to convert lists in Obsidian or elsewhere to lists of cards

### Expected Behavior

One item should be created per input line

### Steps to reproduce

Drag multiple paragraphs, lines, or list items from a web page or Obsidian editor to the add item box

### Operating system

Windows

**Comment by pjeby** [comment]
This appears to be a deliberate change, but no setting was introduced for the behavior?  The old behavior was by design because it was useful for importing, not because multiline cards weren't available.  If this behavior is kept and isn't optional, there should be some way to split an item on line breaks so you can get the same result without tons of editing or having to select and drag single items over and over.

**Comment by mgmeyers** [comment]
@pjeby I've added an option to the card menu to split a multi-line card into multiple cards.

<img width="332" alt="Screen Shot 2021-09-28 at 1 13 53 PM" src="https://user-images.githubusercontent.com/2694747/135158968-89a5d09d-9a2d-4a2a-befc-20ac52afab3e.png">

I know it's not as seamless as it was before, but does this work for you? I tried adding `alt + enter` as another way of doing this, but it seems Obsidian captures that particular key combo.
